sdk-full/src/main/java/module-info.java

```diff
@@ -0,0 +1,22 @@
+module com.hedera.hashgraph.sdk.full {
+    requires transitive com.google.common;
+    requires transitive com.google.protobuf;
+    requires transitive headlong;
+    requires transitive io.grpc.stub;
+    requires transitive io.grpc;
+    requires transitive org.bouncycastle.provider;
+    requires transitive org.slf4j;
```

Comment:
Why those are transitive?

Reply:
See comment below. 

This is an almost exact copy of the file in `sdk` except for exchanging 
`requires grpc.protobuf.lite;` and `requires grpc.protobuf;`

The dependency scope check will make sure that you do not forget to change dependencies in both files if you do in the future. It will do the same analysis for both.

Reply:
My latest comment below affects this conversation as well.

Reply:
Correponding change done here as well.